onebusaway-android/src/androidTest/java/org/onebusaway/android/provider/test/ProviderTest.java

```diff
@@ -60,26 +60,26 @@ public void testStops() {
                 new String[]{ObaContract.Stops._ID, ObaContract.Stops.DIRECTION},
                 null, null, null);
         assertNotNull(c);
-        assertTrue(c.getCount() == 1);
+        assertEquals(1, c.getCount());
         c.moveToNext();
-        assertEquals(c.getString(0), stopId);
+        assertEquals(stopId, c.getString(0));
         c.close();
 
         // Test counting
         c = cr.query(ObaContract.Stops.CONTENT_URI,
                 new String[]{ObaContract.Stops._COUNT},
                 null, null, null);
         assertNotNull(c);
-        assertEquals(c.getCount(), 1);
```

Comment:
@barbeau this was the assertion that was failing. It was 2 instead of one.

Reply:
Whoops. Accidentally commented as my work account ;)

Reply:
Huh.  Not sure why this is working on my device but not yours.  I'm fine with changing it as you have.